package.json

```diff
@@ -110,6 +110,9 @@
     "test-ts": "yarn jest --config jest.config.ts.mjs",
     "test-types": "yarn test-ts --selectProjects type-tests",
     "test": "yarn lint && yarn jest",
+    "typecheck": "yarn typecheck:examples && yarn typecheck:tests",
+    "typecheck:examples": "tsc -p examples/angular --noEmit && tsc -p examples/expect-extend --noEmit && tsc -p examples/typescript --noEmit",
```

Comment:
Using `-p` here, because `--noEmit` is not allow with `-b`. And it does not look right to have `noEmit` in example configs.

Reply:
Can we use `-b` instead and pass all projects at once?

Reply:
In `-b something --noEmit` it is not allow to have `--noEmit`. Or you mean I include add `noEmit: true` to `tsconfig`s of examples? It felt strange to have it there, because that is not what one would do in real world.